windows_performance_counters/README.md

```diff
@@ -21,6 +21,8 @@ No additional installation is needed on your server.
 
 2. [Restart the Agent][5].
 
+**Note**: This check is case sensitive.
```

Comment:
what is case sensitive?

Reply:
Recommend moving the note after configuration, see above comment.
```suggestion
```